command/ibft/candidates/result.go

```diff
@@ -7,28 +7,36 @@ import (
 	"github.com/0xPolygon/polygon-edge/command/helper"
 	ibftHelper "github.com/0xPolygon/polygon-edge/command/ibft/helper"
 	ibftOp "github.com/0xPolygon/polygon-edge/consensus/ibft/proto"
+	"github.com/0xPolygon/polygon-edge/helper/hex"
 )
 
 type IBFTCandidate struct {
-	Address string          `json:"address"`
-	Vote    ibftHelper.Vote `json:"vote"`
+	Address      string          `json:"address"`
+	BLSPublicKey *string         `json:"bls_pubkey"`
+	Vote         ibftHelper.Vote `json:"vote"`
 }
 
 type IBFTCandidatesResult struct {
 	Candidates []IBFTCandidate `json:"candidates"`
 }
 
-func newIBFTCandidatesResult(resp *ibftOp.CandidatesResp) *IBFTCandidatesResult {
+func newIBFTCandidatesResult(resp *ibftOp.CandidatesResp) (*IBFTCandidatesResult, error) {
```

Comment:
Method signature returns an error, but there is no error flow in the code?

Reply:
Removed at be8d9e6a203e0eb6b9555e547519e41a7203aa48